### PR TITLE
Feature/2762 hardcoded mhd conn tmo

### PR DESCRIPTION
**Pull request by kzangeli**
Fixes issue #2762.
CLI parameter for the connection timeout for MHD (-connTimeout).

### CODE REVIEW

File: doc/manuals/admin/cli.md [hunk 1]
```diff
@@ -111,6 +111,8 @@ The list of available options is the following:
     broker process.
 -   **-httpTimeout <interval>**. Specifies the timeout in milliseconds
     for forwarding messages and for notifications.
+-   **-connTimeout <interval>**. Specifies the timeout in seconds
```
Comment (fgalan) [inline doc/manuals/admin/cli.md]: Apart from this description in cli.md, this parameter probably deserves a new entry in https://fiware-orion.readthedocs.io/en/master/admin/perf_tuning/index.html#http-server-tuning.

File: CHANGES_NEXT_RELEASE [hunk 2]
```diff
@@ -1,4 +1,5 @@
 - Fix: libmicrohttpd now uses poll()/epoll() and not select() so fds greater than 1023 can be used for incoming connections (#2724, #2166)
+- Add: new option to set the timeout for REST request: -connTimeout (#2762)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: What about `-reqTimeout`?  To keep the same prefix than in `-reqPoolSize`...

File: doc/manuals/admin/cli.md (hunk 1 above)
Comment (kzangeli) [inline doc/manuals/admin/cli.md]: Fixed in e5a1885

File: doc/manuals/admin/perf_tuning.md [hunk 3]
```diff
@@ -129,6 +129,9 @@ following CLI parameters (see details in the corresponding document):
   thread pool is used (`poll()`). Some performance information regarding this can be found in [the documentation of the
   HTTP server library itself](https://www.gnu.org/software/libmicrohttpd/manual/libmicrohttpd.html#Thread-modes-and-event-loops).
 
+* **reqTimeout**. The inactivity timeout in seconds before a connection is closed. Default value is 10 seconds.
+  To disable this timeout and have the broker wait forever, use a value of 0 for this CLI parameter.
+
 Given that thread creation and destruction are costly operations, it is recommend to use `-reqPoolSize` in
 high load scenarios. In particular, according to [MHD feedback](http://lists.gnu.org/archive/html/libmicrohttpd/2016-12/msg00023.html),
 the pool should be sized with a value equal or close to number of available CPU cores. If you set `-reqPoolSize` to a value higher than
```
Comment (kzangeli) [inline doc/manuals/admin/perf_tuning.md]: ok :-D

Comment (kzangeli) [inline doc/manuals/admin/perf_tuning.md]: Fixed in 9125883

**Comment by fgalan** [comment]
LGTM

**Comment by crbrox** [comment]
LGTM

**Pull request merged**